app/models/lexus_nexus/crime.rb

```diff
@@ -0,0 +1,10 @@
+class LexusNexus::Crime < ApplicationRecord
```

Comment:
Going to adjust this namespace to be independent of Lexus Nexus since we may or may not get to use that data.